ocaml/xapi/xapi_globs.ml

```diff
@@ -1013,7 +1013,7 @@ let xapi_globs_spec =
   ; ("max_active_sr_scans", Int max_active_sr_scans)
   ; ("winbind_debug_level", Int winbind_debug_level)
   ; ("winbind_cache_time", Int winbind_cache_time)
-  ; ("winbind_machine_pwd_timeout", Int winbind_machine_pwd_timeout)
+  ; ("winbind_machine_pwd_timeout", Float winbind_machine_pwd_timeout)
```

Comment:
This will likely need a XenRT test that sets this to a few minutes to check that it works, we're unlikely to have a testcase that runs for >7 days to find a bug in machine password rotation.

Reply:
yes, I agree.

Reply:
BTW, I found each `net changetrustpw` will override the `/var/lib/samba/lock/smb_krb5/krb5.conf.{DOMAIN}`, which have domains KDC, and I think Winbind will use this KDC. 
Thus, the KDC updated the machine password  (with latest Machine password) will always be used.